### DISCUSSION TITLE
Couple of minor bugs

### DISCUSSION THREAD

**Discussion by jh86** [isid0re/SoloLeveling#383] [Q&A]
I've recently been leveling a myriad of characters since USWEST was having some server problems. It was a fun opportunity to observe and manage some different builds and playstyles! A couple of observations where I had to intervene:

1. Amazon (using spec Javazon)
She started off using a bow which is understandable as a levelling route. At level 24 I noticed a disparity between her level and the other profiles running. I saw that she had sold her bow and was using a white club. Her main attack was set to explosive arrow and all her gear was broken (unable to attack at all or even punch out the little fallen). She was out of gold, no mercenary and was going to recoup gold in coldplains or blackmarsh. I ended up playing her manually for a bit in order to get her back on track. I threw on a white bow, got arrows and fixed the gear. I set her back off again with a mercenary and about 30k in the bank. 

When I checked on her again awhile later, she was level 25 and he respecced over to Javelins. Her javelins were broken as well as the rest of her gear. The repair cost for her stealth was 17,500 gold (approximately). I had a though that when she was low on gold, at a weak point in her leveling (trying to run chaos) she may be trying to repair all, but was being unable to do so because of the high cost of the breastplate. The javelins are cheap to repair or replace so I am wondering if there's a way to prioritize single repairing the weapon, especially for a javazon, instead of the blanket repair all. I expect if that was the case, even if her stuff broke and she went on a recoup mission it would have been successful. I ended up playing chaos sanctuary manually, found an ort and then made her an ancients pledge which she had stored a base for in the bank. After that she quickly progressed and has had zero issues moving into late nightmare.

2. Barbarian (using spec Frenzy)
He has been fun to watch, using the warcry/merc combo to progress. On one occasion, in Nightmare I observed him stuck at his stash. The bank had gotten a little full of stuff and the inventory was a bit full too. There were items in an unorganized fashion so I went ahead and moved a couple things around and then he went right back to work. I am not sure exactly the loop he got stuck in but he basically wanted to put his item in the inventory into his bank but even though there was room, the arrangement was off and he didn't know what to do.

I noticed him again in Nightmare, trying with futility to smash the compelling orb. It took me about 2 minutes to realize it was because he was dual wielding and had not removed his secondary weapon in order to progress the quest. I did so manually and he moved along. About an hour later, I checked in again. He was standing at his Nightmare hellforge and again, was stuck not smashing the soulstone due to dualwielding. I have no idea how he got through normal? Maybe he was wearing a shield at the time. Anyways, I have been overall very happy, this has been a delight but I wanted to bring forth these experiences for everyone's benefit. Much love,
jh86.

**Comment by isid0re** [comment]
Just updated the amazon leveling build for a smoother transition. Thanks for posting.
